library/ssl_ciphersuites.c

```diff
@@ -1876,7 +1876,21 @@ int mbedtls_ssl_get_ciphersuite_id( const char *ciphersuite_name )
 
 size_t mbedtls_ssl_ciphersuite_get_cipher_key_bitlen( const mbedtls_ssl_ciphersuite_t *info )
 {
-#if defined(MBEDTLS_CIPHER_C)
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+    psa_key_type_t key_type;
+    psa_algorithm_t alg;
+    size_t key_bits;
+
+    status = mbedtls_ssl_cipher_to_psa( info->cipher,
+                        info->flags & MBEDTLS_CIPHERSUITE_SHORT_TAG ? 8 : 16,
+                        &alg, &key_type, &key_bits );
+
+    if( status != PSA_SUCCESS )
+        return 0;
+
+    return key_bits;
+#elif defined(MBEDTLS_CIPHER_C)
```

Comment:
I think that should be just `#else` because currently TLS depends on `CIPHER_C` and in the future hopefully it will depend on `USE_PSA_CRYPTO || CIPHER_C`, so there's no need to check for `CIPHER_C` here.

(Arguably the existing guard was questionable, but when this function was introduced we knew we were going to rework it soon, so I wasn't too picky. But after this PR no rework is in the pipes for a long time, so I'm being more picky now.)